src/s3_api.cc

```diff
@@ -71,6 +74,18 @@ S3API::Shutdown()
 		}
 
 		Aws::ShutdownAPI(options);
+
+		this->region.clear();
+		this->bucket.clear();
+		this->profile.clear();
+		this->endpoint.clear();
+		// NOTE: logLevel should be reset to match
+		// whatever S3_DEFAULT_LOG_LEVEL is if it changes
```

Comment:
Is this a TODO or a note? It does not look like you are doing what it is recommending

Reply:
It's a reminder to update this if the default changes in the future. I can't use the default that asbackup configs use because it is a different type